README.md

```diff
@@ -55,6 +55,17 @@ For example:
 katex.render("c = \\pm\\sqrt{a^2 + b^2}", element, { displayMode: true });
 ```
 
+Note that the `throwOnError` option only deals with how TeX related parsing errors are handled.
```

Comment:
That's less precise than it could be: the option only deals with unsupported commands, i.e. command names we know nothing about. Other invalid input, like e.g. mismatched braces, will cause exceptions as well no matter this setting. Actually the option description from the list of options above this part is correct in this respect.